raiden/tests/unit/test_service.py

```diff
@@ -60,58 +57,19 @@ def test_ping_unreachable(raiden_network):
     messages = setup_messages_cb()
     UnreliableTransport.network.counter = 0
 
-    ping = Ping(nonce=0)
-    app0.raiden.sign(ping)
-    async_result = app0.raiden.protocol.send_async(
+    ping_message = Ping(nonce=0)
+    app0.raiden.sign(ping_message)
+    ping_encoded = ping_message.encode()
+
+    async_result = app0.raiden.protocol.send_raw_with_result(
+        ping_encoded,
         app1.raiden.address,
-        ping,
     )
 
-    assert async_result.wait(2) is None, "the message was dropped, it can't be acknowledged"
+    assert async_result.wait(2) is None, "The message was dropped, it can't be acknowledged"
 
     for message in messages:
-        assert decode(message) == ping
-
-
-@pytest.mark.parametrize('privatekey_seed', ['ping_dropped_message:{}'])
-@pytest.mark.parametrize('blockchain_type', ['tester'])
-@pytest.mark.parametrize('number_of_nodes', [2])
-@pytest.mark.parametrize('transport_class', [UnreliableTransport])
-def test_ping_ordering(raiden_network):
```

Comment:
This test doesn`t make sense, there is no guarantee about the ordering of Ping messages. Also, the test is written assuming the test is controlling all Ping messages, but these messages are *automatically* used by the healthcheck class and there are more messages than the test is assuming.